### PR TITLE
Add apollo custom extension to replace `apollo_` prefix

### PR DESCRIPTION
**Pull request by yuzushioh**
Added `ApolloExtension` and replace `apollo_` prefix by `apollo.` ref: #1170

### CODE REVIEW

**Comment by designatednerd** [comment]
Looks like you'll need to update the code generation script in `SwiftScripts` to get this to compile: 

Screen Shot 2020-05-02 at 11 25 52 AM

**Comment by yuzushioh** [comment]
@designatednerd  I think its fixed now

**Comment by yuzushioh** [comment]
Ok it's not fixed. I will check it tomorrow.

File: Sources/ApolloCodegenLib/ApolloExtension.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public struct ApolloExtension<Base> {
+  public let base: Base
+}
+
+protocol ApolloCompatible {
```
Comment (designatednerd) [inline Sources/ApolloCodegenLib/ApolloExtension.swift:7]: I suspect this needs to be `public protocol ApolloCompatible`

Comment (yuzushioh) [reply Sources/ApolloCodegenLib/ApolloExtension.swift]: I fixed it. Some of CI tests are still failing but Im not sure why.

Comment (yuzushioh) [reply Sources/ApolloCodegenLib/ApolloExtension.swift]: oh wait Swift Code Generation failure is my fault.

**Comment by yuzushioh** [comment]
I will re-submit the PR after I fix the test

**Pull request closed without merging**